### ISSUE TITLE
Scrolling in the live demo on mac os x with touchpad is bad

### ISSUE THREAD

**Issue by StephanBijzitter** [IjzerenHein/famous-flex-truesize-layoutcontroller-demo#1]
Regarding https://github.com/IjzerenHein/famous-flex/issues/22 the scrolling has the same behaviour as the issue mentioned there.

**Comment by IjzerenHein** [comment]
It scrolls fine on my MacBook air I just tested it with. What browser are you using? Can you record a screen recording (using Quicktime) to demonstrate the problem?

**Comment by StephanBijzitter** [comment]
All browsers below are current releases:
- Opera: choppy and sluggish
- Chrome: choppy
- Firefox: smooth but sluggish
- Safari: smooth

What I noticed when recording was quite weird, it seems that the greater the browser's height, the choppier it gets. When maximising the browser you could see a huge decrease in performance.

https://www.youtube.com/watch?v=XguKBZCvJcU

**Comment by IjzerenHein** [comment]
what macbook model do you have?

**Comment by StephanBijzitter** [comment]
MacBook Pro (Retina, 13-inch, Late 2013)
Processor 2,4 GHz Intel Core i5
Memory 8 GB 1600 MHz DDR3
Graphics Intel Iris 1536 MB
Running OS X Yosemite 10.10.1

**Comment by IjzerenHein** [comment]
Ah I've seen this issue before. This concerns an issue in the browser which causes performance degradation when a div uses 'overflow: hidden'. 
In this demo I had enabled `useContainer` and this causes the performance degradation in the browser. I think that what happens is that when using 'overflow: hidden', the setMatrix operations are no longer GPU accelerated.
I've modified the demo so that it no longer uses `useContainer` but merely `autoPipeEvents` and this solves the problem. Another alternative it so use `useContainer` but remove the `overflow: hidden`, like this:

``` javascript
var scrollView = new FlexScrollView({
    useContainer: true,
   container: {}
});
```
